### ISSUE TITLE
Enabling "Include in Full DPS" breaks the gem comparison window

### ISSUE THREAD

**Issue by raphop** [PathOfBuildingCommunity/PathOfBuilding#5906]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Check for support

- [X] I've checked that the behaviour is supposed to be supported. If it isn't please open a feature request instead (Red text is a feature request).

### What is the behaviour in-game?

N/A

### What is the behaviour in Path of Building?

When turning on "Include in full DPS" the "new support gem window" comparison seems to compare the main skill to the extra skill enabled.

With Orb of Storms enabled:
![image](https://user-images.githubusercontent.com/38050692/229640309-483ec334-8492-4652-ab06-d20c9331946b.png)

With Orb of Storms disabled:
![image](https://user-images.githubusercontent.com/38050692/229640401-539767b1-4b47-4160-9445-952d4f769372.png)

### How to reproduce the issue

- Create a new character
- Add a Main skill and one set to "Include in full DPS"
- With the Main skill selected add the Stun support (Since Stun support shouldn't alter anything it's easy to see the bug)
- Notice the wild comparison numbers

Can also import the attached pob and enable the "Enabling this bugs out the Gem comparison window" socket group and see the bug happening with the main skill 

### Character build code

```shell
https://pobb.in/YXM10r2l6M09
```

### Screenshots

Tested also on a new POB. Arc included in full dps:
![image](https://user-images.githubusercontent.com/38050692/229642281-aa8d0e8b-ca96-4ea9-b322-8f3b8c43e306.png)

Arc not included:
![image](https://user-images.githubusercontent.com/38050692/229642329-12c55a11-fa68-4813-a690-61b53e869df9.png)

**Comment by raphop** [comment]
Side note: This only affects the new support gem comparison, the actual numbers when adding the skill are correct

**Comment by bucketcat** [comment]
This still happens for me, for items, gems and tree. The only way to resolve it is to untick fulldps on every skill. Even if you're sorting effective dps and have the specific selected skill set as combined or effective dps, it still gives negative damage.

**Comment by Jewelots** [comment]
I also have this issue, my Full DPS calculations are broken in a way that they didn't used to be, in that adding any new gem shows the gem's affect as removing 100% of full DPS, I can upload a repro character too if needed.

**Comment by LocalIdentity** [comment]
@Jewelots yes please upload a character as it makes it much easier to test and fix
